ehsm-service/napiparams.js

```diff
@@ -0,0 +1,119 @@
+const keyspec= {
+  type : 'const',
+  arr: ['EHM_AES_GCM_128','EHM_RSA_3072'],
```

Comment:
please compares the arguments definitions, the name not match
EHM_AES_GCM_128','EHM_RSA_3072'  -> EH_AES_GCM_128','EH_RSA_3072'

and later it may support more keypsec, such as :
EH_AES_GCM_256
EH_RSA_2048
...

Reply:
the definitions is in ehsm_provider.h

Reply:
It will be fix.